server/spec/mutations/grid_services/start_spec.rb

```diff
@@ -29,10 +29,8 @@
       redis_service.containers.create!(name: 'redis-1', container_id: '34')
       subject = described_class.new(grid_service: redis_service)
       expect(subject).to receive(:start_service_instances).and_raise(StandardError.new('error'))
-      outcome = subject.run
-      expect(outcome.success?).to be_truthy
       expect {
-        outcome.result.value
+        outcome = subject.run
       }.to raise_exception(StandardError)
```

Comment:
These specs are not that great with the `AsyncMock`, because the `Mutation::Command#run` does not actually ever raise this `StandardError`.